scripts/babel-plugin-remove-log.js

```diff
@@ -0,0 +1,21 @@
+const logMethods = ['log', 'deprecated', 'info', 'group', 'groupCollapsed', 'groupEnd', 'table'];
```

Comment:
@tsherif Eventually this should be moved to luma repo's dev-modules

Reply:
Why not move it to probe.gl? After all it is complementary to the probe.gl log class...